torchgeo/datasets/idtrees.py

```diff
@@ -296,6 +296,7 @@ def _load_boxes(self, path: str) -> Tensor:
                 boxes.append([xmin, ymin, xmax, ymax])
 
         tensor = torch.tensor(boxes)
+        tensor = torch.clamp(tensor, min=0, max=self.image_size[0])
```

Comment:
This assumes that images are always square (i.e. `self.image_size[0] == self.image_size[1]`). Is this always the case for this IDTReeS dataset?

Thinking about whether [`torchvision.ops.clip_boxes_to_image`](https://pytorch.org/vision/0.13/generated/torchvision.ops.clip_boxes_to_image.html#torchvision.ops.clip_boxes_to_image) would work here, or if it's better to keep things simple.

Reply:
Yes, if you check load_image we read them in as 200x200.

I'm indifferent on using clip_boxes_to_image since it appears to perform the same functionality. I would just make sure it's not a recent addition to torchvision so it makes our dependencies strict.

Reply:
Cool, if the images are always square, then this addition is ok. Just hope the next person adding an object detection dataset is careful when copying this code :laughing:

Reply:
I also had the same question about square vs rectangular, can you add a comment saying the image will always be square?